airflow/api_fastapi/core_api/routes/public/dag_parsing.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import TYPE_CHECKING, Annotated
+
+from fastapi import Depends, HTTPException, Request, status
+from fastapi.responses import Response
+from itsdangerous import BadSignature, URLSafeSerializer
+from sqlalchemy import exc, select
+from sqlalchemy.orm import Session
+
+from airflow.api_fastapi.common.db.common import get_session
+from airflow.api_fastapi.common.router import AirflowRouter
+from airflow.api_fastapi.core_api.openapi.exceptions import create_openapi_http_exception_doc
+from airflow.auth.managers.models.resource_details import DagDetails
+from airflow.models.dag import DagModel
+from airflow.models.dagbag import DagPriorityParsingRequest
+
+if TYPE_CHECKING:
+    from airflow.auth.managers.models.batch_apis import IsAuthorizedDagRequest
+
+dag_parsing_router = AirflowRouter(tags=["DAG Parsing"], prefix="/parseDagFile/{file_token}")
+
+
+@dag_parsing_router.put(
+    "",
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
+)
+def reparse_dag_file(
+    file_token: str,
+    session: Annotated[Session, Depends(get_session)],
+    request: Request,
+) -> Response:
```

Comment:
201 are a specific case.
```suggestion
):
```

Reply:
Maybe 
```suggestion
) -> None:
```
Works, but not sure.

Reply:
Modified the code.